nautobot/extras/filters/customfields.py

```diff
@@ -47,7 +47,7 @@ def filter(self, qs, value):
         # Return custom fields that don't match the value and null custom fields
         if self.exclude:
             qs_null_custom_fields = qs.filter(**{f"{self.field_name}__isnull": True}).distinct()
-            return super().filter(qs, value) | qs_null_custom_fields
+            return super().filter(qs, value).distinct() | qs_null_custom_fields
```

Comment:
Had to make this distinct because without it, it caused this error.
![Screenshot 2023-04-13 at 5 49 17 PM](https://user-images.githubusercontent.com/94907097/231887305-716fc893-986f-40dd-ba60-684599e84eb7.png)